.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Package
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  publish:
+    if: github.repository == 'vitest-dev/vitest'
+    runs-on: ubuntu-latest
+    environment: Release
```

Comment:
@patak-dev do you know if it will still have access to `GITHUB_TOKEN` if environment is custom? Does it have access only to the environment secrets or also to organization ones?

Reply:
GITHUB_ env should be available